article/sections/_5_graph_testing.tex

```diff
@@ -1,3 +1,230 @@
 \section{Testing of Causal Graphs}
+\label{sec:graph-testing}
 
-\blindtext[2]
\ No newline at end of file
+\subsection{Description}
+\label{sec:testing-description}
+
+\subsubsection{Testing observable assumptions}
+\label{sec:observable-tests}
+In the last section, we reviewed a process for creating an initial causal graph using expert opinion.
+Critically, after drafting a causal graph, we should immediately test it against empirical data.
+This is important because, if our graph captures inaccurate assumptions about the data generating process, then we have no reason to think that our conclusions from using the graph will be accurate.
+
+To test our causal graphs against data, we will first test the implications of our graph that involve observable variables only.
+We will defer the task of testing implications that involve unobserved / latent variables to later in this subsection.
+For now, recall our discussion in Section \ref{sec:graph-overview} about the two basic implications of causal graphs: marginal independence and conditional independence.
+In both cases, direct testing of marginal or conditional independence amongst nodes in the causal graph may be difficult.
+Indeed, there are no direct tests of conditional independence that can detect all types of dependence, especially for continuous variables \citep{bergsma_2004_testing, shah_2020_hardness}.
+
+As a result of this hardness, there are a myriad of research efforts aimed at testing conditional independence of two variables $X$ and $Y$, given a third variable $Z$ and any number of assumptions about the variables or the test statistic itself.
```

Comment:
Hmm, this sentence made sense to me because if something is hard, then I expect there to be many attacks at the problem (because none are completely successful), hence a myriad of research efforts.

It's def true that the papers I cite are recent research efforts. It's just not clear to me that the recency is a result of the hardness. I'm worried about your suggestion leading some readers to believe that the hardness led to the recency of efforts as opposed to the hardness leading to the many efforts.

Reply:
Fair argument. We can drop it.